lib/managers/account.ts

```diff
@@ -325,7 +340,14 @@ class Account {
   /**
    * Triggered when the user is logged in.
    */
-  private _onRegistered = (): void => {
+  private _onRegistered = (res: any): void => {
+    // Parse the response to get the JWT expiry in epoch
+    // below is the example to get basic 120 sec expiry from response
+    // To do : This needs to be changed in case of login through access Token method
+    if (this.cs.isAccessToken) {
+      const expiryTimeInEpoch = Number(res['response']['headers']['Contact'][0]['parsed']['_parameters']['expires']);
```

Comment:
setting expiry from contact header? then why do we need extra header from server side?